core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -93,6 +99,8 @@
   private ClientMasterSync mClientMasterSync;
 
   private final String mAppId;
+  @GuardedBy("CONTEXT_CACHE_LOCK")
+  private final AtomicInteger mRefCount;
```

Comment:
since this is guarded by `CONTEXT_CACHE_LOCK`, could we use a regular `int` instead?

Reply:
Good point, updated.